specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-07-10/service.json

```diff
@@ -8273,6 +8289,10 @@
         "recoveryReplicaDiskAccountType": {
           "description": "The replica disk type before failover.",
           "type": "string"
+        },
+        "diskEncryptionInfo": {
+          "$ref": "#/definitions/DiskEncryptionInfo",
+          "description": "The recovery disk encryption information (for one / single pass flows)."
```

Comment:
@yaohaizh - Azure Site Recovery (ASR) service supports multiple replication providers and following are published providers (under ReplicationProviderSpecificSettings):									 										 											* A2AReplicationDetails 								 										 										 												* HyperVReplicaAzureReplicationDetails 								 										 										 												* HyperVReplicaBlueReplicationDetails 								 										 										 												* HyperVReplicaReplicationDetails 								 										 										 												* InMageAzureV2ReplicationDetails 								 										 										 												* InMageReplicationDetails

* All the resource operations examples are with one such provider "HyperVReplicaAzure".
* And the new properties belong to "A2A".

We cannot update the examples.

Reply:
For documentation purposes, you should include examples for each of these.

Reply:
@ryansbenson:
1) Are you suggesting us to include examples for all the above shared provider specific properties? If so, we would like to take as a sperate checkin.
PS: The new properties in this PR belong to one provider (Azure To Azure) and have no relation with the provider (HyperV to Azure) mentioned in the examples.
2) Will it be okay if we give examples for all the providers and all the properties - it will be a very very big thing - we have multiple properties (multiple provider specific properties with almost all operations). Please share some complex service examples for our reference & next steps.
3) Few of the properties are mutual exclusive, do we need to give multiple examples within a provider for all the mutual exclusive properties?
4) Is there anyone who can review in IDC (or IDC time friendly office hours)? So that we can expedite the review.